renovate.json

```diff
@@ -1,5 +1,8 @@
 {
   "extends": ["github>redhat-developer/app-services-renovate-config"],
+  "dependencyDashboardApproval": true,
```

Comment:
Setting this as `true` will prevent renovate from creating any PR automatically.
Maybe you want to set it only for major updates?

Anyway, I think it's a matter of deciding how to deal with majors. We could keep the current config and leave PRs like the Jest one open until they become ready to be merged (when the related packages are updated too). 
Otherwise, we could decide to manually trigger the PR creation for every major release, like you propose.

Maybe the second approach is more work? Meaning monitoring periodically the dashboard and manually trigger PRs